package-lock.json

```diff
@@ -489,9 +489,9 @@
       "dev": true
     },
     "aws4": {
-      "version": "1.6.0",
-      "resolved": "https://registry.npmjs.org/aws4/-/aws4-1.6.0.tgz",
-      "integrity": "sha1-g+9cqGCysy5KDe7e6MdxudtXRx4=",
+      "version": "1.7.0",
```

Comment:
why is `package-lock.json` updated in this PR? There are no `package.json` changes as I see.

Reply:
I needed to upgrade google-auth-library to 1.4 to use a few new features in the samples, and it was stuck on 1.3.2 due to the lockfile.